### DISCUSSION TITLE
Versions of the Exp 03

### DISCUSSION THREAD

**Discussion by leodium** [cwmonkey/nms-expeditions#3] [Q&A] [answered]
Hi ...
The Exp03 is available in three versions ...

- normal version "SeasonId": 3
- Redux version "SeasonId": 7
- Redux 'bis' version for this end-of-year holiday in 2023 "SeasonId": 22

I see that you're using the latest version, but you should know that while the missions are identical between SeasonId 7 and SeasonId 22, the rewards are completely different. SeasonId 22 allows you to collect rewards from other seasons. But I think you've already noticed that ...

Is it a choice to only offer the ReduxBis or can we also offer the Redux SeasonId 7? Maybe some people will want to redo SeasonId 7 for specific items that can be collected via the Nexus Expeditions desktop? I've left SeasonId 7 and 22 in my Github for those ...

Thanks for this amazing works !

**Comment by cwmonkey** [comment, marked as answer]
Ah, ok I was confused. I thought 3b offered the old AND new rewards, so I was trying to simplify. Instead of only offering 3b, I will only offer 3 (7). If they want the other rewards they can play the other expeditions, or use your JSON :)

**Comment by cwmonkey** [comment]
Changed to season 7 instead of 22 in dcd4cb95c1f559cfc1cc33fea1e176f3b169f340
